index.html

```diff
@@ -2298,7 +2556,9 @@ <h3>
         </h3>
         <p>
           The <dfn>application's name</dfn> is derived from either the
-          [=manifest/name=] member or [=manifest/short_name=] member.
+          [=manifest/name=] member or [=manifest/short_name=] member. The user
```

Comment:
This needs more clarification as to which one wins.

Reply:
In Chromium we use them depending on UX needs. It's not implemented, but I would like to follow similar rules for extensions - `short_name` is truncated to 12 characters and `name` is truncated to 75.

This sets expectations for devs, and allows the user agent to show an app name where there might not be much space.

Reply:
Could we discuss that in a separate issue? It used to be like this and is not directly related to l10n.

Reply:
How they are treated by the OS is implementation specific. I don't think we need to say anything here.

Reply:
IDK if it's true that these are only used by the OS - the user agent uses these strings too. Happy for it to be a non-normative note - truncating characters is actually fraught with weirdness due to multi-character sequences now etc, so it would be helpful to document here limits to prevent those breakages.

User agents can't just allow any string here to be displayed in full, as it'll cause weird UX issues with dialogs / menu items / etc. This was the purpose of, say, `short_name`.

But - I certainly don't want to block this. And perhaps you are right here @christianliebel that this can be addressed in a separate issue / pull request.

Reply:
I'll second @dmurph's _truncating characters is actually fraught with weirdness_. 

Example. 🏴󠁧󠁢󠁳󠁣󠁴󠁿

The Scottish flag emoji is 7 code points. The JavaScript string length is 14. It's one grapheme (user perceived character, i.e. screen position). If the `short_name` limit is `short_name.length < 13`, the flag won't fit...

As a reminder, I18N's [guidelines about text truncation](https://www.w3.org/TR/international-specs/#char_truncation) apply to length limits. One has to be careful about what one is counting (bytes, code points, graphemes) to ensure that some languages are not disadvantaged because of how their writing system works (emoji makes for a great demo, but there are languages that use combining marks similar to the way emoji sequences work that have the same counting issue)

Reply:
Created https://github.com/w3c/manifest/issues/1145 for further discussion.